frontend/public/components/start-guide.tsx

```diff
@@ -52,11 +51,10 @@ export const withStartGuide = (WrappedComponent, disable: boolean = true) =>
     FLAGS.SHOW_OPENSHIFT_START_GUIDE,
     FLAGS.CAN_CREATE_PROJECT,
   )(({ flags, ...rest }: any) => {
-    const { kindObj } = rest;
-    const kind = _.get(kindObj, 'kind', rest.kind);
+    const { kindObj = {} } = rest;
```

Comment:
The logic here is different than before. Previously we were defaulting to `rest.kind` if `kindObj` was not set.

I also wouldn't default it to `{}`. That's not a valid `kindObj`. Better to leave it `const { kindObj } = rest;` and always use optional chaining.